src/java.base/share/classes/java/util/WeakHashMap.java

```diff
@@ -238,6 +286,15 @@ public WeakHashMap() {
         this(DEFAULT_INITIAL_CAPACITY, DEFAULT_LOAD_FACTOR);
     }
 
+    /**
+     * Constructs a new, empty {@code WeakHashMap} with the {@link ValuePolicy},
+     * the default initial capacity (16) and load factor (0.75).
+     * @param  valuePolicy     The {@link ValuePolicy} for keys that are Value objects; non-null
```

Comment:
This should include `@throws NullPointerException`:
```suggestion
     *
     * @param  valuePolicy     The {@link ValuePolicy} for keys that are Value objects
     * @throws NullPointerException if {@code valuePolicy} is null
```